src/Libraries/RevitNodes/Elements/Parameter.cs

```diff
@@ -40,5 +205,190 @@ public override string ToString()
             }
             return string.Format("{0} : {1}", Name, value);
         }
+
+        #endregion
+
+        #region Shared Parameters
+
+        /// <summary>
+        /// Get shared parameter file path
```

Comment:
can you explain the logic behind this to users? something created by default etc?